blackbox/docs/sql/reference/select.txt

```diff
@@ -263,9 +263,18 @@ rows. You can find :ref:`here <sql_union>` sample usage of UNION ALL.
 
 .. note::
 
-    ORDER BY, LIMIT and OFFSET can be attached only to the last select statement
-    of the UNION ALL and will apply to the whole result of the UNION. Column names
-    used in ORDER BY must refer to the outputs of the 1st SELECT statement.
+    ORDER BY, LIMIT and OFFSET can be applied only after the last SELECT
+    statement of the UNION ALL and will apply to the whole result of the UNION.
+    In order to apply ORDER BY and/or LIMIT and/or OFFSET to any of the partial
+    SELECTS those selects should become a subquery.
+
+.. note::
+
+    Column names used in ORDER BY must be position numbers or refer to the
```

Comment:
not sure if it would need to say "positional numbers"

Reply:
thought would seem that you can only refer to columns otherwise.